Tools/scripts/build-jsbsim.sh

```diff
@@ -6,16 +6,20 @@ echo "---------- $0 start ----------"
 
 # don't waste time rebuilding jsbsim if we already have a working copy of it
 # this can save ~10 or minutes on some hardware
-JSBBINARY=~/jsbsim/src/JSBSim
+JSBBINARY=~/jsbsim/build/src/JSBSim
 if [ -f $JSBBINARY ]; then
     echo "$JSBBINARY exists, skipping build. (to force rebuild then remove this file and re-run) "
 else
     echo "$JSBBINARY does not exist, building it in your home folder:"
     cd ~ 
     rm -rf jsbsim
-    git clone https://github.com/tridge/jsbsim.git
+    git clone git://jsbsim.git.sourceforge.net/gitroot/jsbsim/jsbsim
+    sudo apt-get install cmake
```

Comment:
The current version doesn't install anything.  I'd rather the build fall over with an error instead of modifying people's build environment in a script labelled "build".